cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/LogMessage.java

```diff
@@ -123,14 +125,14 @@ public void prepareForDeferredProcessing() {
     if (args != null && argumentArray == null) {
       argumentArray = new Object[args.length];
       for (int i = 0; i < args.length; i++) {
-        argumentArray[i] = args[i].toString();
+        argumentArray[i] = Objects.toString(args[i]);
```

Comment:
Prefer to use the `java.util.Objects` instead.

Reply:
This is a good catch

Reply:
Sure, changed to fully qualified import